packages/data-service-generator/src/server/resource/dto/create-dto-module.ts

```diff
@@ -8,6 +8,7 @@ import {
   addImports,
   importContainedIdentifiers,
```

Comment:
please remove the import for the old `importContainedIdentifiers` as it's currently not in use